src/commonMain/kotlin/exchange.dydx.abacus/processor/router/skip/SkipProcessor.kt

```diff
@@ -27,6 +28,13 @@ internal class SkipProcessor(
 
     var skipTokens: Map<String, Map<String, List<Map<String, Any>>>>? = null
     override var exchangeDestinationChainId: String? = null
+    override var cosmosWalletConnected: Boolean? = false 
```

Comment:
[suggestion] we'll have phantom wallet integrated soon enough, can we create an enum to describe all the possible wallets instead? then we can do something like `ALOWED_CHAIN_TYPES + SELECTED_WALLET_CHAIN_TYPE` on line 306 to make this easily extendable to new wallets

Reply:
I can take a stab